portfolio/src/main/webapp/script.js

```diff
@@ -13,8 +13,14 @@
 // limitations under the License.
 
 function fetchfunction(){
-    fetch("/data").then(response => response.json()).then((json) => {
-        document.getElementById('testing').innerText = json;
+    let fullComments = []
+    let comments = "";
+    fetch("/data").then(response => response.json()).then((tasks) => {
+        fullComments.push(tasks);
+        for (i=0; i<fullComments[0].length; i++) {
+            comments = comments + fullComments[0][i]["propertyMap"]["comment"] + "\n"; 
```

Comment:
I add an if statement for this but have yet to add an action statement for it

Reply:
I think it's probably fine to have no action for if it is null. If a comment is null, we most likely don't want to display it anyway (because it's likely that something went wrong). So just skipping it should be sufficient. In which case, you can shorten this by having a single else that checks for the negative case (rather than the positive case like you have right now) and then move the logic in the else block to the if block and skip having an else block all together. However, the way you have it now is good too because logging weird responses can absolutely be helpful.